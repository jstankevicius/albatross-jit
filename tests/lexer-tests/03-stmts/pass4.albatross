if (0) { repeat(0) { return 0; } }
repeat (0) { while (0) { 0+0; } otherwise { 0-0; } }