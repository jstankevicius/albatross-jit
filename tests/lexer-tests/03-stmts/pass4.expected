1 1 IF
4 1 (
5 1 INT 0
6 1 )
8 1 {
10 1 REPEAT
16 1 (
17 1 INT 0
18 1 )
20 1 {
22 1 RETURN
29 1 INT 0
30 1 SEMICOLON
32 1 }
34 1 }
1 2 REPEAT
8 2 (
9 2 INT 0
10 2 )
12 2 {
14 2 WHILE
20 2 (
21 2 INT 0
22 2 )
24 2 {
26 2 INT 0
27 2 PLUS
28 2 INT 0
29 2 SEMICOLON
31 2 }
33 2 OTHERWISE
43 2 {
45 2 INT 0
46 2 MINUS
47 2 INT 0
48 2 SEMICOLON
50 2 }
52 2 }
